MediaBrowser.Model/Dto/UserDataDto.cs

```diff
@@ -0,0 +1,40 @@
+namespace MediaBrowser.Model.Dto
+{
+    /// <summary>
+    /// Class UserDataDto extends UserItemDataDto to allow nullable members.
+    /// This change allow us to implement the new /Users/{UserId}/Items/{ItemId}/UserData endpoint.
+    /// This object allows the requestor to update all or specific user data fields without altering the non-nullable members state.
```

Comment:
This summary becomes the description for the API specification, we don't need developer notes for that. What benefit does this description have for client developers that use a generated SDK/apiclient?

Reply:
Updated to reflect the requested changes.